documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1931,31 +1938,105 @@ endif::community[]
 
 // Type: procedure
 // ModuleID: setting-up-postgresql-permissions-required-by-debezium-connectors
-// Title: Setting up PostgreSQL permissions required by {prodname} connectors
+// Title: Setting up PostgreSQL permissions for the {prodname} connector
 [[postgresql-permissions]]
 === Setting up permissions
 
-Setting up a PostgreSQL server to run a {prodname} connector requires a database user who can perform replications. Replication can be performed only by a database user who has appropriate permissions and only for a configured number of hosts. Also, you must configure the PostgreSQL server to allow replication to take place between the server machine and the host on which the PostgreSQL connector is running.
+Setting up a PostgreSQL server to run a {prodname} connector requires a database user who can perform replications.
```

Comment:
interesting. I would have used 'that' instead of 'who'. database users aren't people

Reply:
Or as the saying goes, "Veeeery Interesting. But stupid!" :) Good catch, @davecramer. Thanks.

Clearly, I was losing context, and focused only on user in the typical sense vs. the Debezium user. I'll effect a repair.